console/src/main/webapp/manager/app/components/orgs/orgs.tpl.html

```diff
@@ -34,7 +34,7 @@ <h3 class="roles-title">
       </thead>
 
       <tbody>
-        <tr dir-paginate="org in orgs.orgs | filter:orgs.q | itemsPerPage: orgs.itemsPerPage">
+        <tr dir-paginate="org in orgs.orgs | filter:orgs.q | filter: {pending: orgs.org=='pending'} | itemsPerPage: orgs.itemsPerPage">
```

Comment:
maybe `===` ??

Reply:
Please, could you more specific @RemiDesgrange about your request ?